src/libsyntax/parse/mod.rs

```diff
@@ -253,9 +253,10 @@ pub fn file_to_filemap(sess: &ParseSess, path: &Path, spanopt: Option<Span>)
     let bytes = match File::open(path).read_to_end() {
         Ok(bytes) => bytes,
         Err(e) => {
-            err(&format!("couldn't read {:?}: {:?}",
+            let error_msg = e.desc;
+            err(&format!("couldn't read {:?}: {}",
                         path.display(),
-                        e)[]);
+                        error_msg)[]);
```

Comment:
I think this should actually continue to format `e` entirely, it should just use `{}` instead of `{:?}`